src/matter/cluster/Cluster.ts

```diff
@@ -48,5 +50,47 @@ export const OptionalEvent = <FT extends FieldTemplates>(id: number, priority: E
 export interface Attributes { [key: string]: Attribute<any> }
 export interface Commands { [key: string]: Command<any, any> }
 export interface Events { [key: string]: Event<any> }
-export interface Cluster<C extends Commands, A extends Attributes, E extends Events> { id: number, name: string, commands: C, attributes: A, events: E }
-export const Cluster = <C extends Commands, A extends Attributes, E extends Events>({id, name, attributes = <A>{}, commands = <C>{}, events = <E>{} }: { id: number, name: string, attributes?: A, commands?: C, events?: E } ):Cluster<C, A, E> => ({ id, name, commands, attributes, events });
+
+export type GlobalAttributes<F extends BitTemplates> = {
```

Comment:
If you want to add referecne use  Core Specs 7.13

Reply:
Added